recipes-kernel/linux-firmware/linux-firmware_git.bb

```diff
@@ -160,6 +160,9 @@ do_install() {
 	# Remove pointless bash script
 	rm ${D}/lib/firmware/configure
 
+	# Remove python script used to check the WHENCE file
+	rm ${D}/lib/firmware/check_whence.py
```

Comment:
That is from an upstream commit: [oe-core/1fc4d5a31f](https://git.openembedded.org/openembedded-core/commit/?h=master-next&id=1fc4d5a31f05970d8d80b0106ea81d486f298e33), please mention the source.

Side-note: upstream release branches use fixed revisions for the `linux-firmware_%.bb`. If that is acceptable, using these should be considered. In general, considering OpenXT now closely follows Yocto releases, `linux` components could use the upstream recipes: Dunfell is on Linux 5.4, Zeus on Linux 4.19, so branches `zeus` and `master` (to be `dunfell`) are already aligned. It would also let us use defconfig configuration fragments.